README.md

```diff
@@ -38,6 +37,6 @@ And you should see an image such as below if you select no blur and 'use_backgro
 
 ## Support
 * [Documentation](https://alwaysai.co/docs/)
-* [Community Discord](https://discord.gg/alwaysai)
+* [Community Discord](https://discord.gg/z3t9pea)
```

Comment:
On our Discord, it looked like https://discord.gg/alwaysai was still a valid reference link -- is this not working for you or is there another reason to change it?